src/site/_includes/js/eligibility.js

```diff
@@ -2693,6 +2708,28 @@ function clipperStartResult(input) {
   return program.getResult();
 }
 
+function homelessPreventionSystemResult(input) {
+  const grossLimit = MonthlyIncomeLimits.fromAnnual(
+    cnst.homelessPreventionSystem.ANNUAL_INCOME_LIMITS,
+    cnst.homelessPreventionSystem.ANNUAL_INCOME_LIMIT_ADDL_PERSON);
```

Comment:
I don't think `cnst.homelessPreventionSystem.ANNUAL_INCOME_LIMIT_ADDL_PERSON` is defined